.github/workflows/ontobot.yaml

```diff
@@ -79,3 +67,4 @@ jobs:
           title: ${{ env.PR_TITLE }}
           base: ${{ github.head_ref }}
           branch: ${{ env.branch-name }}
+          token: ${{ secrets.GH_TOKEN }}
```

Comment:
@matentzn , when the token is passed here, it authorizes this workflow to kickstart any qc workflow(s) for the repository.

Reply:
Very good to know, thanks!